examples/sensor-samplingrate/src/components/SensorSamplingUpdate.jsx

```diff
@@ -0,0 +1,257 @@
+
+  updateSamplingRateStatus = () => {
+    const { sensor, status } = this.state;
+    const {
+      data,
+      astarte,
+      samplingRateInterface,
+      refreshSamplingRate
+    } = this.props;
+    if (sensor && status) {
+      this.setState({ loading: true });
+      astarte
+        .setSensorSamplingById({
+          id: data.id,
+          interfaces: samplingRateInterface,
+          sensor_id: sensor,
+          key: "enable",
+          unset: status === "unset",
+          params: {
+            data: status === ENABLE
+          }
+        })
+        .then(() => {
+          refreshSamplingRate(data.id, samplingRateInterface);
+          this.setState({ loading: false });
+        })
+        .catch(err => this.handleError(err));
+    }
+  };
+
+  render() {
+    const { sensorValues } = this.props;
+    const {
+      loading,
+      visible,
+      current,
+      status,
+      samplingRateStatus
+    } = this.state;
+    return (
+      <Col xs={12} className="border-separate">
+        <div className="sensor-sampling-div">
+          <Row className="main-row col-sm-9 align-items-center mx-auto my-3">
+            <Col sm={3} xs={12} className={"p-0"}>
+              <span>Select Sensor:</span>
+            </Col>
+            <Col sm={9} xs={12}>
+              <FormGroup>
+                <FormControl
+                  as="select"
+                  name="sensor"
+                  required
+                  onChange={this.handleSensor}
+                >
+                  {Object.keys(sensorValues).map((item, index) => {
+                    return (
+                      <option selected={!index} key={index}>
+                        {item}
+                      </option>
+                    );
+                  })}
+                </FormControl>
+              </FormGroup>
+            </Col>
+          </Row>
+          <Row className="main-row col-sm-9 align-items-center mx-auto my-3">
+            <Col sm={3} xs={12} className={"p-0"}>
+              <span>Sampling Status:</span>
+            </Col>
+            <Col sm={9} xs={12}>
+              <FormGroup>
+                <FormCheck
+                  onChange={this.handleChange}
+                  value="1"
+                  name="status"
+                  inline
+                  label="On"
+                  type="radio"
+                />
+                <FormCheck
+                  onChange={this.handleChange}
+                  value="0"
+                  name="status"
+                  inline
+                  label="Off"
+                  type="radio"
+                />
+                <FormCheck
+                  onChange={this.handleChange}
+                  value="unset"
+                  name="status"
+                  inline
+                  label="Handled by device"
+                  type="radio"
+                />
+              </FormGroup>
+            </Col>
+          </Row>
+          {status === ENABLE ? (
```

Comment:
why assigning to a constant when you can just compare the string itself?

status === "enable". Is someone forcing you to use the "1" string?

Reply:
Actually i was using "1" for "On", "0" for "Off" and "unset" for "Handled By device"
in radio fields value. thats why matching with "1" string

I'll change field value now, "enable", "disable" or "unset"